test_coverage/test_gcovr.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# for info see 'read.me'
+
+PREFIX_ERRN="ERRN:"
+DIR_BUILD="build_gcovr"
+TIMESTAMP=$(date +"%y-%m-%d-%H:%M:%S")
+DIR_REPORT="test_coverage/report_gcovr_${TIMESTAMP}"
+DIR_HOME=$PWD
+DIR_CI="ci/travis"
+INDEX_REPORT="report.html"
+CTEST_REPORT="ctest.log"
+
+# Installing gtest
+if [ ! -d ../$DIR_CI/googletest ]
+  then
+    cd ../$DIR_CI
+    # Setup test
+    . setup_gtest.sh
+    export CMAKE_SYSTEM_PREFIX_PATH=$PWD/googletest/install
+    cd $DIR_HOME
+fi
+# Jump to build folder
+if [ ! -d ../$DIR_BUILD ]
+  then
+    mkdir ../$DIR_BUILD
+    if [ $? -ne 0 ]
+    then
+      echo $PREFIX_ERRN "cannot make dir '"$DIR_BUILD"'"
+      exit 1
+    fi
+fi
+cd ../$DIR_BUILD
+rm -rf ./*
+# Build facelift with flags
+cmake .. -DCMAKE_BUILD_TYPE=Debug -DFACELIFT_ENABLE_GCOV=ON -DFACELIFT_BUILD_TESTS=ON -DCMAKE_SYSTEM_PREFIX_PATH=$CMAKE_SYSTEM_PREFIX_PATH && make
```

Comment:
can't you simply set CMAKE_CXX_FLAGS here and remove all your changes to CMakeLists.txt ?

Reply:
Sure